src/main/java/com/easypost/model/beta/Referral.java

```diff
@@ -153,10 +174,62 @@ public static CreditCard addCreditCard(String referralApiKey, String number, int
      * @param priority        Priority of this credit card.
      * @return CreditCard object.
      * @throws Exception when the request fails.
+     * @deprecated Use {@link #addCreditCardToUser(String, String, int, int, String, PaymentMethod.Priority)} instead.
      */
     public static CreditCard addCreditCard(String referralApiKey, String number, int expirationMonth,
                                            int expirationYear, String cvc, CreditCardPriority priority)
             throws Exception {
+
+        // Convert a CreditCardPriority enum to a PaymentMethod.Priority enum
+        PaymentMethod.Priority priorityEnum = null;
+        switch (priority) {
+            case PRIMARY:
+                priorityEnum = PaymentMethod.Priority.PRIMARY;
+                break;
+            case SECONDARY:
+                priorityEnum = PaymentMethod.Priority.SECONDARY;
+                break;
+            default:
+                break;
+        }
+
+        if (priorityEnum == null) {
+            throw new Exception("Invalid credit card priority.");
+        }
+
+        PaymentMethodObject paymentMethodObject =
+                addCreditCardToUser(referralApiKey, number, expirationMonth, expirationYear, cvc, priorityEnum);
+
+        // Convert the new PaymentMethodObject back into the deprecated CreditCard object
+
```

Comment:
Nit: no line breaks between a comment and its associated code, leads to confusion.

Reply:
Resolved